tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -49,63 +52,162 @@ class InternalCallMapHandlerTest extends TestCase
      * @var array<int|string, string|list<string>>
      */
     private static $ignoredFunctions = [
```

Comment:
These lists were generated so might need to update local environment to catch extensions that were added/supported.